packages/web-client/app/components/workflow-thread/index.ts

```diff
@@ -12,12 +12,19 @@ interface WorkflowThreadArgs {
 
 export default class WorkflowThread extends Component<WorkflowThreadArgs> {
   threadEl: HTMLElement | undefined;
-  workflow = new AnimatedWorkflow(this.args.workflow);
+  @cached
```

Comment:
I didn't realise how many times this property was being read until I removed `@cached`. Did you happen to look into why it is sometimes read incessantly, @lukemelia ?

Reply:
I did not. I just knew that the logic wouldn't work if we were constructing new AnimatedWorkflow instances that maintain their own state.